### ISSUE TITLE
still overwriting my users, when there's no change

### ISSUE THREAD

**Issue by theoneandonly-vector** [myOmikron/mailcow-ldap-sync#6]
I run the command like this:
```
cd /opt/mailcow-ldap-sync/mailcow_ldap_sync && /opt/mailcow-ldap-sync/venv/bin/python3 ./main.py --update-only-on-change-by-ldap
```
I get this
log, but I'm 100% sure there was no change at all:
```
INFO:mailcow_ldap_sync:{'username': 'administrator@mydomain.tld', 'active': 1, 'active_int': 1, 'domain': 'mydomain.tld', 'relayhost': None, 'name': 'Administrator Administrator', 'local_part': 'administrator', '>
INFO:mailcow_ldap_sync:{'username': 'administrator@mydomain.tld', 'active': 1, 'active_int': 1, 'domain': 'mydomain.tld', 'relayhost': None, 'name': 'Administrator Administrator', 'local_part': 'administrator', '>
INFO:mailcow_ldap_sync:LDAP user uid=Administrator,cn=users,dc=mydomain,dc=tld was modified in mailcow
```

additional question.. when this works correctly and not overwrite the users on each run, will it still overwrite them when there is a pw-change form LDAP-side (needed..)?

**Comment by theoneandonly-vector** [comment]
something is weird.
I don't see a difference in my logs "mailcow_ldap_sync.log" if I run it manually or when crontab runs the script. but somehow it wants to reauth me in SoGo when I run it manually, but when crontab runs the same script it doesn't.

**Comment by theoneandonly-vector** [comment]
..update.. it happens on crontab also.

**Comment by theoneandonly-vector** [comment]
@myOmikron  do you think this is fixable?

**Comment by myOmikron** [comment]
It surely is, I have currently no spare time to dig into this, but I'll be happy to review and accept Pull Requests

**Comment by theoneandonly-vector** [comment]
is it possible to use the script somehow with passing a specific address? so I could update specific users on demand

**Comment by theoneandonly-vector** [comment]
without destroying all web-sessions

**Comment by myOmikron** [comment]
Implemented with ec1d266534968e8df49ff48bafec6bc881aea5b4

**Comment by theoneandonly-vector** [comment]
when I start the script without changing the filter to only update a specific user but use " --update-only-on-change-by-ldap" it will still re-authenticate all users on sogo, because there is some change done by the script.
It tells me "None" in the terminal when I run it.
